opt_einsum/paths.py

```diff
@@ -1015,7 +1015,10 @@ def __call__(self,
             else:
                 cost_cap = self.cost_cap
             # set the factor to increase the cost by each iteration (ensure > 1)
-            cost_increment = max(min(map(size_dict.__getitem__, subgraph_inds)), 2)
+            if len(subgraph_inds) == 0:
+                cost_increment = 1
```

Comment:
@jcmgray is this value correct?

Reply:
I think shouldn't matter as subgraph will always be size 1? But probably should be `2`, (since its a multiplicative increment 1x does nothing).